backend/crud.py

```
@@ -0,0 +1,245 @@
+        title=spot.title,
+        description=spot.description,
+        image_url=image_url,  # R2からのURLまたはNone
+        crowd_level=spot.crowd_level if spot.crowd_level else models.CrowdLevelEnum.MEDIUM,
+        rating=spot.rating if spot.rating is not None else 3,
+    )
+
+    try:
+        db.add(db_spot)
+        db.flush()  # IDを取得するためflushを実行
+        
+        # 投稿報酬としてコインを付与（同一トランザクション内）
```

Ruff (0.14.7)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)